src/components/Stack/Stack.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { States } from 'storybook-states';
 import { space } from '@moda/tokens';
 import { Stack, StackProps } from './Stack';
 
-export default { title: 'Components|Stack' };
+export default { title: 'Components/Stack' };
 
```

Comment:
can you please change `HorizontalUsingustifyContent` to `HorizontalUsingJustifyContent` on line 48